### PR TITLE
Dissociate python version from test tools & tested version.

### PR DESCRIPTION
**Pull request by ryarnyah**
## Proposed Changes

  - Dissociate python version from test tools & tested version.
  - Add python 3.5 to test suite. 
  - Reactivate support for python2.
  - Add python-xdist for testing
  - Add python-instafail to see fail before testing end
  - Make molecule verbose to see live testing
  - Make all test independent

### CODE REVIEW

File: molecule/default/tests/ansible_utils.py [hunk 1]
```diff
@@ -262,57 +258,57 @@ def call_kafka_acl(
...
+    return call_kafka_acl(host, test_acl_configuration, check)
 
 
 def check_configured_topic(host, topic_configuration,
                            topic_name, kafka_servers, deleted_options=None):
     """
     Test if topic configuration is what was defined
     """
-    # Forcing api_version to 0.11.0 in order to be sure that a
+    # Forcing api_version to 0.31.0 in order to be sure that a
```
Comment (StephenSorriaux) [inline molecule/default/tests/ansible_utils.py:311]:
```suggestion
    # Forcing api_version to 0.11.0 in order to be sure that a
```

File: molecule/default/tests/ansible_utils.py [hunk 2]
```diff
@@ -353,7 +349,7 @@ def check_configured_acl(host, acl_configuration, kafka_servers):
     """
     Test if acl configuration is what was defined
     """
-    # Forcing api_version to 0.11.0 in order to be sure that a
+    # Forcing api_version to 0.31.0 in order to be sure that a
```
Comment (StephenSorriaux) [inline molecule/default/tests/ansible_utils.py:352]:
```suggestion
    # Forcing api_version to 0.11.0 in order to be sure that a
```

**Review by StephenSorriaux — APPROVED** [review]
Love this. Brilliant idea to use an executor with a dedicated Python version to trigger the lib and do everything else with an unique version of Python. Thank you a lot!

File: molecule/default/tests/ansible_utils.py (hunk 1 above)
Comment (ryarnyah) [reply molecule/default/tests/ansible_utils.py]: Done :)

File: molecule/default/tests/ansible_utils.py (hunk 2 above)
Comment (ryarnyah) [reply molecule/default/tests/ansible_utils.py]: Done :)

**Pull request merged**